src/helpers/FileManagement.ts

```diff
@@ -80,4 +80,32 @@ export function inline_regex_target_in_function_of(position: string, columnId: s
             regex_target = contentHasFrontmatter ? `$1\n${columnId}:: ${newValue}$2` : `${columnId}:: ${newValue}\n$1`;
     }
     return regex_target;
-}
\ No newline at end of file
+}
+
+
+export function sanitize_path(path: string, replacement = ''){
+    const illegalCharacters = /[\*"\\\/<>:\|\?]/g
```

Comment:
I really like the idea.
We can move this to the Constants helper to make it reusable and its cool for me